js/actions/actions.js

```diff
@@ -0,0 +1,31 @@
+var AppDispatcher = require('../dispatcher/app_dispatcher');
+var ActionTypes = require('../constants/constants').ActionTypes;
+var UserStore = require('../stores/user_store')
+
+module.exports = {
+  createMessage: function(text) {
+    AppDispatcher.dispatch({
+      type: ActionTypes.NEW_MESSAGE,
+      text: text,
+      user: UserStore.localUser().name,
+      timestamp: new Date().getTime()
```

Comment:
So the facebook examples don't do that... but it does make a certain amount of sense especially if the payloads end up getting generated by separate code. I'm fine with whatever.